services/lesson.js

```diff
@@ -11,6 +11,10 @@ const lessonService = {
     )
 
     return await Lesson.create({ author, title, description, attachments: fileUrls })
+  },
+
+  deleteLesson: async (id) => {
+    await Lesson.findByIdAndRemove(id).exec()
```

Comment:
There should be a check if a current user's I'd is equal to author

Reply:
Fixed